openvdb_points/tools/IndexFilter.h

```diff
@@ -243,6 +244,70 @@ class AttributeHashFilter
 }; // class AttributeHashFilter
 
 
+template <typename LevelSetGridT>
+class LevelSetFilter
+{
+public:
+    typedef typename LevelSetGridT::ValueType ValueT;
+
+    struct Data
+    {
+        Data(const LevelSetGridT& _grid, const math::Transform& _transform,
+             const ValueT _min, const ValueT _max)
+            : accessor(_grid.getConstAccessor())
+            , levelSetTransform(_grid.transform())
+            , transform(_transform)
+            , min(_min)
+            , max(_max) { }
+
+        // not a reference to ensure const-accessor is unique per-thread
+        const typename LevelSetGridT::ConstAccessor accessor;
```

Comment:
In the unit tests below, data is only constructed once, then passed as a reference or pointer everywhere. Are we expecting the user to construct data locally for each thread?